samples/client/petstore/kotlin-jackson/src/main/kotlin/org/openapitools/client/Application.kt

```diff
@@ -10,7 +10,7 @@ fun main() {
     println("<top>.main")
     val inventory = StoreApi().getInventory()
     println("Inventory : $inventory")
-    val pet = Pet(name = "Elliot", photoUrls = listOf<String>("https://jameshooverstudios.com/wp-content/uploads/2015/04/Majestic-Dog-Photography-Elliot-Nov-5-2014.jpg", "https://express-images.franklymedia.com/6616/sites/981/2020/01/22105725/Elliott.jpg").toTypedArray(), id = 123456453, category = Category(id = 13259476, name = "dog"), tags = listOf<Tag>(Tag(id = 194093, name = "Elliot")).toTypedArray(), status = Pet.Status.AVAILABLE)
+    val pet = Pet(name = "Elliot", photoUrls = listOf<String>("https://jameshooverstudios.com/wp-content/uploads/2015/04/Majestic-Dog-Photography-Elliot-Nov-5-2014.jpg", "https://express-images.franklymedia.com/6616/sites/981/2020/01/22105725/Elliott.jpg"), id = 123456453, category = Category(id = 13259476, name = "dog"), tags = listOf<Tag>(Tag(id = 194093, name = "Elliot")), status = Pet.Status.AVAILABLE)
```

Comment:
this class is created not by generator, why it there? may be it should be added to jackson templates?

Reply:
It can be an old file or a file written by hand to test the generated code.

Reply:
what the script I can launch locally for verify tests as on CI?

Reply:
Sorry, but I don't know

Reply:
Better let CI run the tests as many Kotlin samples need to be tested.